code/modules/clothing/masks/gasmask.dm

```diff
@@ -45,6 +96,7 @@
 	visor_flags_inv = HIDEEYES
 	visor_flags_cover = MASKCOVERSEYES
 	resistance_flags = FIRE_PROOF
+	max_filters = 0
```

Comment:
```suggestion
	max_filters = 1
```
desc = "A gas mask with built-in welding goggles and a face shield. Looks like a skull - clearly designed by a nerd."
It's literally a gas mask